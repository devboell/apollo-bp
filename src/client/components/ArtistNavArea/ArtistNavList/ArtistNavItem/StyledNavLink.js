import styled from 'styled-components'
import { NavLink } from 'react-router-dom'


export const activeClassName = 'nav-item-active'

const StyledNavLink = styled(NavLink).attrs({
  activeClassName,
})`
  text-decoration: none;
  > h2 {
    display: inline-block;
    margin: 10px;
    border-bottom: solid transparent 1px;
    color: rgb(222, 81, 11);
  }

  &.${activeClassName} {
    > h2 {
      border-bottom: solid rgb(145, 144, 144) 1px;
    }
  }
`

export default StyledNavLink
