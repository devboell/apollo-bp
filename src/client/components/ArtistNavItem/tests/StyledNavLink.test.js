import React from 'react'
import 'jest-styled-components'

import StyledNavLink from '../StyledNavLink'

it('StyledNavLink, snapshot', () => {
  const wrapper = shallow(<StyledNavLink to="/testPath" />)
  expect(wrapper).toMatchStyledComponentsSnapshot()
})
