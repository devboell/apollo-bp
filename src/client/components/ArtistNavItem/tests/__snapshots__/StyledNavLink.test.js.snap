// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledNavLink, snapshot 1`] = `
.bCihhC {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.bCihhC > h2 {
  display: inline-block;
  margin: 10px;
  border-bottom: solid transparent 1px;
  color: rgb(222, 81, 11);
}

<NavLink
  activeClassName="nav-item-active"
  className="sc-bdVaJa bCihhC"
  to="/testPath"
/>
`;
